src/app/jscaip/Vector.ts

```diff
@@ -13,19 +14,9 @@ export class Vector implements ComparableObject {
                Math.abs(this.y) === length;
     }
     public toMinimalVector(): Vector {
-        const absX: number = Math.abs(this.x);
-        const absY: number = Math.abs(this.y);
-        const maxAbs: number = Math.max(absX, absY);
-        let greatestCommonDivider: number = 1;
-        let divider: number = 2;
-        while (divider <= maxAbs) {
-            if (this.x % divider === 0 && this.y % divider === 0) {
-                greatestCommonDivider = divider;
-            }
-            divider++;
-        }
-        return new Vector(this.x / greatestCommonDivider,
-                          this.y / greatestCommonDivider);
+        let greatestCommonDivisor: number = MathUtils.gcd(this.x, this.y);
```

Comment:
AAAAH ! Mooossieux le fan des const :D 
TPR ! (ça veut dire TU PEUX RIRE HEIN !?)

Reply:
DAMN you are right! J'ai corrigé le let du coup, j'te laisse résoudre !